app/scripts/controllers/network/provider-api-tests/block-param.ts

```diff
@@ -1718,9 +1729,9 @@ export function testsForRpcMethodSupportingBlockParam(
         [
           ['less than the current block number', '0x200'],
           ['equal to the curent block number', '0x100'],
-        ],
+        ] as any,
```

Comment:
This isn't ideal, but was copied from the `core` repo. At some point I'd like to go through those tests again and do some cleanup.